package.json

```diff
@@ -2715,7 +2715,7 @@
         "@types/semver": "^7.3.3",
         "@types/string-replace-webpack-plugin": "^0.1.0",
         "@types/tar": "^4.0.3",
-        "@types/vscode": "1.44.0",
+        "@types/vscode": "1.49.0",
```

Comment:
This PR is not yet complete. This package doesn't exist (yet), and we will also need to update `package-lock.json`.